server/common/HatoholArmPluginInterface.cc

```diff
@@ -157,6 +157,11 @@ struct HatoholArmPluginInterface::Impl {
 		connected = false;
 		hapi->onSessionChanged(NULL);
 	}
+	
+	bool getconnectedstatus(void) {
```

Comment:
Hmm..., Hatohol project uses camelCase naming rule.
Please use `getConnectedStatus` instead of `getconnectedstatus`.